core/src/storage/resident_credentials.py

```
@@ -17,23 +17,53 @@
+
+
+def get_fido2_counter() -> int:
+    return device.get_fido2_counter()
+
+
+def _increase_fido2_counter() -> None:
+    value = device.get_fido2_counter()
+    if value >= MAX_RESIDENT_CREDENTIALS:
+        raise ValueError(
+            f"FIDO2 counter cannot be greater than {MAX_RESIDENT_CREDENTIALS}"
+        )
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)